pkg/controllers/vdb/podfacts.go

```diff
@@ -389,6 +392,7 @@ func (p *PodFacts) collectPodByStsIndex(ctx context.Context, vdb *vapi.VerticaDB
 			return err
 		}
 		pf.hasNMASidecar = vk8s.HasNMAContainer(&pod.Spec)
+		pf.isNMAContainerReady = vk8s.IsNMAContainerReady(pod)
```

Comment:
What about when nma is not running in a separate container?

Reply:
This will return false. Now NMA is always running as a sidecar so I didn't consider that case. 

If we need to cover the case nma is not running in a seprate container, we need to update gatherScript to check if nma process exists. However, that is for the old vertica versions, and it isn't worth doing that because the ReIP error is rare and don't affect anything.